src/domains/sign/io_sign_backoffice_func.tf

```diff
@@ -0,0 +1,141 @@
+locals {
+  backoffice_func_settings = {
+    for s in var.io_sign_backoffice_func.app_settings :
+    s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${module.key_vault.name};SecretName=${s.key_vault_secret_name})" : s.value
+  }
+}
+
+module "io_sign_backoffice_func" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v6.2.1"
+
+  name                = format("%s-backoffice-func", local.project)
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+
+  health_check_path = "/health"
+
+  node_version    = "18"
+  runtime_version = "~4"
+  always_on       = true
+
+  app_settings = local.backoffice_func_settings
+
+  subnet_id = module.io_sign_backoffice_snet.id
+
+  allowed_subnets = [
+    module.io_sign_snet.id
+  ]
+
+  app_service_plan_id = module.io_sign_backoffice_app.plan_id
+
+  application_insights_instrumentation_key = data.azurerm_application_insights.application_insights.instrumentation_key
+  system_identity_enabled                  = true
+
+  tags = var.tags
+}
+
+resource "azurerm_monitor_autoscale_setting" "io_sign_backoffice_func" {
+  count               = var.io_sign_backoffice_func.sku_tier == "PremiumV3" ? 1 : 0
+  name                = format("%s-autoscale", module.io_sign_backoffice_func.name)
+  resource_group_name = azurerm_resource_group.backend_rg.name
+  location            = azurerm_resource_group.backend_rg.location
+  target_resource_id  = module.io_sign_backoffice_func.app_service_plan_id
+
+  profile {
+    name = "default"
+
+    capacity {
+      default = var.io_sign_backoffice_func.autoscale_default
+      minimum = var.io_sign_backoffice_func.autoscale_minimum
+      maximum = var.io_sign_backoffice_func.autoscale_maximum
+    }
+
+    rule {
```

Comment:
@Krusty93 we should track this task: to put the scaling info into the module itself